README.md

```diff
@@ -15,6 +15,214 @@ Build
+
+  var db = new Cliquetis();
+  var articles = db.collection('articles');
+
+```
+
+The collection object has the following attributes:
+
+* **last_modified**: last synchronization timestamp, ``undefined`` if never sync'ed.
+
+> Synchronization timestamps are persisted in the device local storage.
+
+
+### List records
+
+```js
+
+  articles.all().then(result => {
+    {
+      data: [
+        {
+          id: "705b17be-e957-4c14-8f4c-86f8eaac29c0",
+          title: "foo"
+        },
+        {
+          id: "68e63131-3859-40cc-a4f7-b237ca179329",
+          last_modified: 1432222889336,
+          title: "Web page"
+        },
+      ]
+    }
+  });
+
+```
+
+> Records with ``last_modified`` attribute were sync'ed on a server.
+
+
+### Create records
+
+```js
+
+  articles.save({title: "foo"}).then(result => {
+    data: {
+      id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+      title: "foo",
+    }
+  });
+
+```
+
+> Records identifiers are generated locally using UUID4.
+
+
+### Update records
+
+```js
+
+  var existing = {
+    id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+    title: "bar"
+  };
+
+  articles.save(existing).then(result => {
+    data: {
+      id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+      title: "bar",
+    }
+  });
+
+```
+
+### Delete records
+
+```js
+
+articles.delete({
+    selector: {
+      age: {'$min': 42}
+    }
+  }).
+  then(result => {
+    data: [
+      {
+        id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+        deleted: true,
+      }
+    ]
+  });
+
+```
+
+A shortcut for ``articles.delete({id: {'$eq': id}})`` is available at ``articles.delete(id)``.
+
+
+
+### Filter records
+
+```js
+
+articles.all({
+    selector: {
+      title: {'$eq': 'foo'}
+    }
+  }).
+  then(result => {
+    data: [
+      …
+    ]
+  });
+
+```
+
+A shortcut for ``articles.all({id: {'$eq': id}})`` is available at ``articles.get(id)``.
+
+
+**Sorting**
+
+```js
+
+articles.all({
+    sort: [{added_on: 'desc'}]
+  }).
+  then(result => {
+    data: [
+      …
+    ]
+  });
+
+```
+
+
+### Fetch and publish changes
+
+```js
+
+var options = {
+  remote: "https://server/v1/{}/records",
+  headers: {
+    "Authorization": "Basic bWF0Og=="
+  },
+  mode: Cliquet.SAFE,
+};
+
+articles.sync(options).then(result => {
+    {
+      ok: true,
+      created: [],  // missing locally.
+      updated: [],  // changed since last sync.
+      deleted: [],  // deleted since last sync.
+      conflicts: []  // changed both sides.
+    }
+});
+
+```
+
+> If ``ok`` is ``false``, the list of conflicting records will be provided in
+> ``conflicts`` attribute.
+>
+> Conflicts must be resolved locally and ``sync()`` called again.
+
+
+**Options**
+
+* **remote**: Server URI with placeholder for collection name
+* **headers**: HTTP headers (*authentication*)
+* **mode**: ``Cliquet.SAFE`` (*server wins*), ``Cliquet.FORCE`` (*client wins*)
+
+**Synchronization strategy**
+
+# Fetch changes since last synchronization using ``?_since``;
```

Comment:
This is not the right syntax for numeric list.
See http://stackoverflow.com/editing-help#advanced-lists